.github/workflows/build-and-test.yml

```diff
@@ -192,16 +192,8 @@ jobs:
             /home/runner/go/pkg/mod
             /home/runner/.cache/go-build
           key: go-pkg-mod-${{ runner.os }}-${{ hashFiles('**/go.mod', '**/go.sum') }}
-      - name: Cache Tools
-        id: tool-cache
-        uses: actions/cache@v2
-        env:
-          cache-name: cache-tool-binaries
-        with:
-          path: /home/runner/go/bin
-          key: tools-${{ runner.os }}-${{ hashFiles('./internal/tools/go.mod', './cmd/mdatagen/go.mod', './cmd/mdatagen/*.go') }}
       - name: Build Collector for All Architectures
-        run: make -j4 binaries-all-sys
```

Comment:
What was the issue? Using too much memory since the compiler also runs in parallel?

Reply:
Yes, machine too small to be able to compile in parallel